lib/unified_health_data/adapters/clinical_notes_adapter.rb

```diff
@@ -0,0 +1,153 @@
+# frozen_string_literal: true
+
+module UnifiedHealthData
+  module V2
+    module Adapters
+      class ClinicalNotesAdapter
+        LOINC_CODES = {
+          '11506-3' => 'PHYSICIAN_PROCEDURE_NOTE',
+          '11488-4' => 'CONSULT_RESULT',
+          '18842-5' => 'DISCHARGE_SUMMARY'
+        }.freeze
+
+        EMPTY_FIELD = 'None recorded'
```

Comment:
SUGGESTION: Make this an empty string or `nil`, I would rather have nothing denoting an empty field